### ISSUE TITLE
have plain color fill instead of "||||" bars to show mem and cpu

### ISSUE THREAD

**Issue by mrlee14** [ClementTsang/bottom#1434]
### Checklist

- [X] I've looked through [the documentation](https://clementtsang.github.io/bottom/nightly/) and  [existing open issues](https://github.com/ClementTsang/bottom/issues?q=is%3Aopen+is%3Aissue+label%3Afeature) for similar feature requests.

### Describe the feature request

This is only thing which does not digest properly here...

**Comment by ClementTsang** [comment]
Could you please give more information on what you mean? Is this for the basic mode?

**Comment by mrlee14** [comment]
> Could you please give more information on what you mean? Is this for the basic mode?

yes, basic mode graph bars for RAM, CPU,SWAP looks  this way RAM [ ||||||| ] (htop style), its hard to quickly read out actual usages, compared with with rust 'btop' this one has solid color graph but still not perfect (btop uses too much cpu and it seems not maintained)...
